Documentation/Parameters/Resources.md

```diff
@@ -0,0 +1,90 @@
+# Resource Parameters
+
+Resource parameters allow picking of resources in workbooks. This is useful in setting the scope from which to get the data from. An example is allowing users to select the set of VMs which the charts later will use when presenting the data.
+
+Values from resource pickers can come from the workbook context, static list or from Azure Resource Graph queries.
+
+## Creating a resource parameter using workbook resources
+1. Start with an empty workbook in edit mode.
+2. Choose _Add parameters_ from the links within the workbook.
+3. Click on the blue _Add Parameter_ button.
+4. In the new parameter pane that pops up enter:
+    1. Parameter name: `Applications`
+    2. Parameter type: `Resource picker`
+    3. Required: `checked`
+    4. Allow multiple selections: `checked`
+5. Get data from: `Workbook Resources`
+6. Include only resource types: `Application Insights`
+7. Choose 'Save' from the toolbar to create the parameter.
+
+![Image showing the creation of a resource parameter using workbook resources](../Images/Parameters-Resource-Create.png)
+
+## Creating a resource parameter using Azure Resource Graph
```

Comment:
you might want to add a caveat here that Azure Resource Graph is not (currently) in all clouds?

Reply:
and probably a link to their docs as well?  workbooks itself has https://aka.ms/AzureResourceGraph-QueryLanguage